driver.go

```diff
@@ -129,10 +134,7 @@ func NewPodmanDriver(logger hclog.Logger) drivers.DriverPlugin {
 		ctx:            ctx,
 		signalShutdown: cancel,
 		logger:         logger.Named(pluginName),
-		podmanClient: &PodmanClient{
-			ctx:    ctx,
-			logger: logger.Named("podmanClient"),
-		},
+		podmanClient2:  apiclient.NewClient(logger),
```

Comment:
I think it may be nice to keep the logger named as `podman_client` for additional context about who is logging